config-default.js

```diff
@@ -50,7 +50,10 @@ module.exports = {
 
     // the shell to run when spawning a new session (i.e. /usr/local/bin/fish)
     // if left empty, your system's login shell will be used by default
-    shell: ''
+    shell: '',
+
+    // enable audible bell
+    audibleBell: true
```

Comment:
No problem, I think a true/false switch would be intuitive, but a path would be better for future plugins. We could include a comment describing the behavior, how does this look?

```javascript
// set hterm audible bell path, default is 'lib-resource:hterm/audio/bell'
// if left empty, the audible bell will not ring
bellSoundPath: 'lib-resource:hterm/audio/bell',
```